### ISSUE TITLE
no-cyclic-action when dispatch: false 

### ISSUE THREAD

**Issue by craig-dae** [cartant/eslint-plugin-rxjs#57]
The following should not produce a `no-cyclic-action` error, or if I'm wrong, should explain to me the correct way of doing it: 

```
  successfulLoad$ = createEffect(
    () =>
      this.actions$.pipe(
        ofType(GetCurrentUserSuccess),
        tap(() => localStorage.setItem(KEY_FAILED, '0')),
      ),
    { dispatch: false },
  );
```

**Comment by cartant** [comment]
See #54.

**Comment by craig-dae** [comment]
That's helpful. But the main issue with this library is lack of guidance for these issues. Looks like there are two suggested courses of action. 

1. Use the  ` ignoreElements()` operator at the end of the pipe, instead of `{ dispatch: false }`, to avoid ngrx-specific behavior
2.  Ignore the line, which is the author's intention, since their opinion is that `{ dispatch: false }` should be infrequently used

How difficult would it be to add some guidance here? Is this a common trap that people fall into outside of ngrx?

**Comment by cartant** [comment]
As mentioned in [this comment](https://github.com/cartant/eslint-plugin-rxjs/issues/54#issuecomment-730696017), number 2 is not something I'd be prepared to support. I don't want to handle stuff like this:

```ts
createEffect(someEffect, { dispatch: false });
```

and the rule is not NgRx-specific - it will work with redux-observable, too.

My suggestion would be number 1. (Also, IMO, the `{ dispatch: false }` business is a questionable API design.)

I will, however, add something to the docs.

**Comment by craig-dae** [comment]
Thanks. Should I close this, or do you want to keep it open until you update the docs?

**Comment by cartant** [comment]
I'll close it after I've updated the docs.
